.ci/.e2e-tests-for-elastic-agent.yaml

```diff
@@ -3,21 +3,31 @@ PLATFORMS:
   stack:
     image: "ami-0d90bed76900e679a"
     instance_type: "c5.4xlarge"
+    username: admin
```

Comment:
Well done here! As this is the descriptor of the machine, it makes total sense to have the configuration here